plugins/20_events/subcommands/default

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+
+events_main_cmd() {
```

Comment:
Was hoping we'd just call all the functions `main`, but this is certainly useful if we ever wanted to start sourcing the subcommands.